tests/test-libmaven/utilities.h

```diff
@@ -0,0 +1,171 @@
+#ifndef DTUTILS_H
+#define DTUTILS_H
```

Comment:
"utilities.h" is a very general name. Ideally we'd reserve this for some project-wide utilities. Let's call it `testutils.h`? or anything that tells the user that file brings in test fixtures. Also change the include guards to according to the file name.

Reply:
As testutils was similar to the file in maven tests. Therefore, I have named it as testLibmavenUtils

Reply:
um ... testutils.h from MavenTests and testutils.h from test-libmaven will almost never be included in the same namespace. So its okay to name them the same. `testLibmavenUtils` looks ugly :P

Reply:
Okkay, I will make the change